tests/gcp/00-assert.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test
```

Comment:
namespace test not specififed

Reply:
See https://github.com/kartverket/skiperator/pull/70#discussion_r998125265